src/blog/2022/12/flowforge-1-2-0-released.md

```diff
@@ -0,0 +1,85 @@
+---
+title: FlowForge 1.2 is now available with single sign on and persistent context storage
+subtitle: Our final release for 2022 with some great new features to try out
+description: 
+date: 2022-12-22 18:00:00.0
+authors: ["rob-marcer"]
+video: u7TjqUAub1g
+---
+
+Control access to FlowForge using single sign-on and retain context values when restarting projects
+
+<!--more-->
+
+We're pleased to announce version 1.2 is now available! The latest release of the FlowForge application contains new features, improvements, better documentation, and bug fixes.
+
+We've put a great deal of work in this release to make it easier to run your own self-managed instance of FlowForge. That includes significant improvements for FlowForge in [Kubernetes](https://flowforge.com/docs/install/kubernetes/) and [Docker](https://flowforge.com/docs/install/docker/).
+
+Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above.
+
+## Features
+
+- [Single Sign-On](https://github.com/flowforge/flowforge/issues/226) Single sign-on (SSO) is a method of authentication that allows a user to access multiple applications or systems with a single set of login credentials, improving security, productivity, and user experience, and reducing IT overhead. We've implemented SSO using the Security Assertion Markup Language [(SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) framework. This allows users of FlowForge Cloud, Enterprise and Community Edition to easily access their FlowForge projects.
```

Comment:
```suggestion
[Single Sign-On](https://github.com/flowforge/flowforge/issues/226) Single sign-on (SSO) is a method of authentication that allows a user to access multiple applications or systems with a single set of login credentials, improving security, productivity, and user experience, and reducing IT overhead. We've implemented SSO using the Security Assertion Markup Language [(SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) framework. This allows users of FlowForge Cloud, Premium and the open source edition to easily access their FlowForge projects.
```

Reply:
@robmarcer Please tell people to create a support ticket through the contact us page, or support@flowforge.com to enable this for their team on FlowForge cloud. Also link to the admin docs for Premium customers.

Reply:
See also: https://github.com/flowforge/flowforge/blob/main/docs/cloud/README.md#single-sign-on